variables/OneSeoVariable.php

```diff
@@ -60,6 +59,24 @@ public function meta()
             $metaData['url'] = $element->url;
         }
 
+        // Override defaults with template customizations
+        // Title…
+        $dividerChar = ' ' . $pluginSettings->titleDividerCharacter . ' ';
+        $customTemplateTitle = craft()->oneSeo_meta->getMetaTitle();
+        if ($customTemplateTitle) $metaData['pageTitle'] = implode($dividerChar, $customTemplateTitle);
+        // Description…
+        $customTemplateDescription = craft()->oneSeo_meta->getMetaDescription();
+        if ($customTemplateDescription) $metaData['description'] = $customTemplateDescription;
+        // Image…
+        $customTemplateImage = craft()->oneSeo_meta->getMetaImage();
+        if ($customTemplateImage) $metaData['image'] = $customTemplateImage;
```

Comment:
small small suggestion: we could shorten this ever so slightly by:

```
if ($customTemplateImage = craft()->oneSeo_meta->getMetaImage()) {
  $metaData['image'] = $customTemplateImate;
}
```

same for the two above